spring-cloud-modules/pom.xml

```diff
@@ -18,7 +18,7 @@
 
     <modules>
         <module>api-gateway</module>
-	<module>spring-cloud-loadbalancer</module>
```

Comment:
is there an indentation below?
If yes, we don't need it.

Reply:
i fixed it, thanks!